modules/components/pages/outputs/schema_registry.adoc

```diff
@@ -108,17 +111,39 @@ output:
 
 The base URL of the schema registry service.
 
-
 *Type*: `string`
 
-
 === `subject`
 
+The subject name.
+
 This field supports xref:configuration:interpolation.adoc#bloblang-queries[interpolation functions].
 
+*Type*: `string`
+
+=== `backfill_dependencies`
+
+Backfill missing schema references and previous versions.
+
+*Type*: `bool`
+
+*Default*: `true`
+
+=== `translate_ids`
+
+Encodes Redpanda schema IDs ready to publish to a schema registry.
+
+*Type*: `bool`
+
+*Default*: `true`
+
+=== `input_resource`
+
+The label of the xref:components:inputs/schema_registry.adoc[`schema_registry` input] from which to read source schemas.
```

Comment:
We don't mention the need to use a registry_input with this connector anywhere. Should we?

Reply:
Good point! It should be mandatory when `backfill_dependencies: true` (the default). By default, `input_resource: schema_registry_input`, but it will fail if users don't actually configure a `schema_registry` input. Guess we should mention this in the description?

Reply:
Updated